modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx

```diff
@@ -82,6 +113,15 @@ const QuotedSummary = ({externalReferenceCode}: externalReferenceCodeType) => {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+	function handleValuesFormatter(value: string) {
+		if (value === bodilyInjury || motoristBodilyInjury) {
+			return value.replaceAll(',000', ',000.00');
+		}
+		else {
+			return value.replace(',000', ',000.00');
+		}
```

Comment:
Please, remove the ```else``` condition.

```suggestion
		if (value === bodilyInjury || motoristBodilyInjury) {
			return value.replaceAll(',000', ',000.00');
		}
		
		return value.replace(',000', ',000.00');
```

Reply:
Done.